sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/checkin/SentryCheckIn.java

```diff
@@ -0,0 +1,41 @@
+package io.sentry.spring.jakarta.checkin;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.jetbrains.annotations.ApiStatus;
+import org.springframework.core.annotation.AliasFor;
+
+/** Sends a {@link io.sentry.CheckIn} for the annotated method. */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
```

Comment:
Does it make sense to apply this `Annotation` on types? This would lead to checkins for every method in the class.
If that's what we want, then we need to adapt the `SentryCheckInAdvice` class to also check for the annotation on the class level (similar to `SentryTransactionAdvice`).

also applies to non-jakarta

Reply:
No, forgot to remote the target on the annotation.